cni/network/network_test.go

```diff
@@ -1371,6 +1371,31 @@ func TestPluginSwiftV2MultipleAddDelete(t *testing.T) {
 			wantErr:    false,
 			wantNumEps: 2,
 		},
+		{
+			name: "SwiftV2 Add Infra and InfiniteBand",
```

Comment:
can we add for all possible nic combinations?

Reply:
we have all combinations in this ut:
infraNIC + delegatedNIC
infraNIC + IBNIC
two delegatedNICs